### PR TITLE
Added text if you need access to service

### PR DESCRIPTION
**Pull request by pierskarsenbarg**
For example, if you don't have external internet access

### CODE REVIEW

File: themes/default/content/docs/troubleshooting/_index.md [hunk 1]
```diff
@@ -526,3 +526,11 @@ There are two ways to fix this, one way if you have access to an Intel based com
 ### I'm seeing other errors, please help
 
 Get in touch with us, either through the [Community Slack](https://slack.pulumi.com) or by emailing our support team: [support@pulumi.com](mailto:support@pulumi.com).
+
+## I can't connect to the Pulumi Service
```
Comment (susanev) [inline themes/default/content/docs/troubleshooting/_index.md:530]: could we maybe work network into the heading?

File: themes/default/content/docs/troubleshooting/_index.md [hunk 2]
```diff
@@ -526,3 +526,11 @@ There are two ways to fix this, one way if you have access to an Intel based com
...
+
+If your network is locked down to external traffic, you're using the service to manage your state, and your security team need to give you access to the service then here's the information you can pass over.
```
Comment (susanev) [inline themes/default/content/docs/troubleshooting/_index.md:532]:
```suggestion
If your network is locked down to external traffic, you're using the service to manage your state, and your security team needs to give you access to the service then here's the information you can pass over.
```

**Review by susanev — APPROVED** [review]

File: themes/default/content/docs/troubleshooting/_index.md [hunk 3]
```diff
@@ -526,3 +526,11 @@ There are two ways to fix this, one way if you have access to an Intel based com
 ### I'm seeing other errors, please help
 
 Get in touch with us, either through the [Community Slack](https://slack.pulumi.com) or by emailing our support team: [support@pulumi.com](mailto:support@pulumi.com).
+
+## I can't connect to the Pulumi Service
+
+If your network is locked down to external traffic, you're using the service to manage your state, and your security team needs to give you access to the service then here's the information you can pass over.
```
Comment (cnunciato) [inline themes/default/content/docs/troubleshooting/_index.md:532]:
```suggestion
If your network blocks external traffic and you're using the Pulumi Service to manage your state, your security team may need the following details to allow the Pulumi CLI to connect to the Service:

 * The URL that the Pulumi CLI uses to connect to the Service is `https://api.pulumi.com`. (It does not use `https://app.pulumi.com`, so if you want to view the console, you'll need enable that as well.)
 * All access goes over HTTP via port 443.
```

**Review by cnunciato — CHANGES_REQUESTED** [review]
Thanks for adding this! Some minor suggestions but otherwise 👍.

File: themes/default/content/docs/troubleshooting/_index.md [hunk 4]
```diff
@@ -526,3 +526,11 @@ There are two ways to fix this, one way if you have access to an Intel based com
...
+
+The URL that the Pulumi CLI connects to is: `https://api.pulumi.com` (not `https://app.pulumi.com`, although if you want to view the console then you'll need access to that as well).
+
+All access goes over HTTPS via port 443.
```
Comment (cnunciato) [inline themes/default/content/docs/troubleshooting/_index.md:536]: If your network blocks external traffic and you're using the Pulumi Service to manage your state, your security team may need the following details to allow the Pulumi CLI to connect to the Service:

 * The URL that the Pulumi CLI uses to connect to the Service is `https://api.pulumi.com`. (It does not use `https://app.pulumi.com`, so if you want to view the console, you'll need enable that as well.)
 * All access goes over HTTP via port 443.

Comment (cnunciato) [inline themes/default/content/docs/troubleshooting/_index.md:536]: If your network blocks external traffic and you're using the Pulumi Service to manage your state, your security team may need the following details to allow the Pulumi CLI to connect to the Service:

 * The URL that the Pulumi CLI uses to connect to the Service is `https://api.pulumi.com`. (It does not use `https://app.pulumi.com`, so if you want to view the console, you'll need enable that as well.)
 * All access goes over HTTP via port 443.

**Review by cnunciato — CHANGES_REQUESTED** [review]
Let's try this again. 🤞

File: themes/default/content/docs/troubleshooting/_index.md (hunk 4 above)
Comment (cnunciato) [inline themes/default/content/docs/troubleshooting/_index.md:536]:
```suggestion
If your network blocks external traffic and you're using the Pulumi Service to manage your state, your security team may need the following details to allow the Pulumi CLI to connect to the Service:

 * The URL that the Pulumi CLI uses to connect to the Service is `https://api.pulumi.com`. (It does not use `https://app.pulumi.com`, so if you want to view the console, you'll need enable that as well.)
 * All access goes over HTTP via port 443.
```

File: themes/default/content/docs/troubleshooting/_index.md [hunk 5]
```diff
@@ -526,3 +526,10 @@ There are two ways to fix this, one way if you have access to an Intel based com
 ### I'm seeing other errors, please help
 
 Get in touch with us, either through the [Community Slack](https://slack.pulumi.com) or by emailing our support team: [support@pulumi.com](mailto:support@pulumi.com).
+
+## I can't connect to the Pulumi Service
+
+If your network blocks external traffic and you're using the Pulumi Service to manage your state, your security team may need the following details to allow the Pulumi CLI to connect to the Service:
+
+ * The URL that the Pulumi CLI uses to connect to the Service is `https://api.pulumi.com`. (It does not use `https://app.pulumi.com`, so if you want to view the console, you'll need enable that as well.)
+ * All access goes over HTTPS via port 443.
```
Comment (susanev) [inline themes/default/content/docs/troubleshooting/_index.md:535]:
```suggestion
* The URL that the Pulumi CLI uses to connect to the Service is `https://api.pulumi.com`. (It does not use `https://app.pulumi.com`, so if you want to view the console, you'll need enable that as well.)
* All access goes over HTTPS via port 443.
```

**Review by cnunciato — APPROVED** [review]

**Pull request merged**